src/main/java/calculator/view/CalculatorView.java

```diff
@@ -0,0 +1,7 @@
+package calculator.view;
+
+public interface CalculatorView {
+
+    String displayInput();
```

Comment:
```suggestion
    String getInput();
```
과 같은 형식으로 메소드의 확실한 목적성을 보여주는 것이 좋을 듯합니다. 이는 현재 input을 출력하는 형태처럼 읽혀집니다.

Reply:
아... 확실히 그런 부분을 고려하면 `displayInput` 대신 `displayInputView`로 메서드의 이름을 변경하는 것이 더 나았을 것 같네요..